consensus/istanbul/backend/backend.go

```diff
@@ -481,6 +487,19 @@ func (sb *Backend) HasBadProposal(hash common.Hash) bool {
 	return sb.hasBadBlock(hash)
 }
 
+func (sb *Backend) IsKnownMessage(msg istanbul.Message) bool {
```

Comment:
This is no longer used. Should it be removed?

Reply:
Why is it no longer used?

Reply:
I removed the check against known messages (perhaps incorrectly) when adding round change messages to the round change message set in `handleRoundChangeCertificate`